src/main/io/serial_1wire.c

```diff
@@ -154,104 +130,87 @@ static void gpioSetOne(uint32_t escIndex, GPIO_Mode mode) {
 #define ESC_OUTPUT(escIndex)   gpioSetOne(escIndex, Mode_Out_PP)
 #endif
 
-#if defined(STM32F3DISCOVERY)
-#define LED_PRGMR_RX GPIO_Pin_8
-#define LED_PRGMR_TX GPIO_Pin_10
-// Top Left LD4, PE8 (blue)-- from programmer (RX)
-#define RX_LED_OFF GPIOE->BRR =  LED_PRGMR_RX
-#define RX_LED_ON  GPIOE->BSRR = LED_PRGMR_RX
-// Top Right LD5, PE10 (orange) -- to programmer (TX)
-#define TX_LED_OFF GPIOE->BRR =  LED_PRGMR_TX
-#define TX_LED_ON  GPIOE->BSRR = LED_PRGMR_TX
-static void ledInitDebug(void)
-{
-  uint32_t pinmask = LED_PRGMR_RX|LED_PRGMR_TX;
-  GPIO_DeInit(GPIOE);
-  RCC_AHBPeriphClockCmd(RCC_AHBPeriph_GPIOE, ENABLE);
-  gpio_set_mode(GPIOE, pinmask, Mode_Out_PP);
-  GPIOE->BRR = pinmask;
-}
-#else
 #define RX_LED_OFF LED0_OFF
 #define RX_LED_ON LED0_ON
 #define TX_LED_OFF LED1_OFF
 #define TX_LED_ON LED1_ON
-#endif
 
 // This method translates 2 wires (a tx and rx line) to 1 wire, by letting the
 // RX line control when data should be read or written from the single line
 void usb1WirePassthrough(uint8_t escIndex)
 {
-#ifdef STM32F3DISCOVERY
-  ledInitDebug();
+#ifdef BEEPER
+    // fix for buzzer often starts beeping continuously when the ESCs are read
+    // switch beeper silent here
+    beeperSilence();
```

Comment:
ledInitDebug() does not make sense for me. LED0 and LED1 are assigned for the board in target.h